deployment_report/model/utils/test/test_config_util.py

```diff
@@ -32,11 +32,8 @@ def test_get_cadence_version(no_ingress_simulation_fixture: conftest.DSA):
     # get the resource cache
     resource_cache: Dict = no_ingress_simulation_fixture.resource_cache()
 
-    # get the ConfigMaps
-    config_maps: Dict = resource_cache[ResourceTypeValues.K8S_CORE_CONFIG_MAPS][ITEMS_KEY]
-
     # test the util method
-    assert config_util.get_cadence_version(config_maps=config_maps) == KubectlTest.Values.CADENCEINFO
+    assert config_util.get_cadence_version(resource=resource_cache) == KubectlTest.Values.CADENCEINFO
```

Comment:
Refactoring the param name (suggested in the config_util.py changes) to `resource_cache` should change this call.